### ISSUE TITLE
Currently Wayland is not supported(Ubuntu 18.04)

### ISSUE THREAD

**Issue by Edwardzcn** [kbumsik/VirtScreen#49]
I have already installed the software. But I can not start it with the warning message "Currently Wayland is not supported".

Laptop Specs:

- UBUNTU 18.04.3 
- Processor: Intel® Core™ i7-7700HQ CPU @ 2.80GHz × 8
- Graphics: Intel® HD Graphics 630 (Kaby Lake GT2)
- GNOME: 3.28.2
- OS type: 64-bit

Tablet Specs:
- iPad Pro 2018

VNC Viewer app by RealVNC (https://www.realvnc.com/en/connect/download/viewer/ios/)
What I tried so far:
- Just follow the README

```bash
sudo apt-get install virtscreen
sudo dpkg ...
```

I have already seen the icons. But whatever I click the icon or just type the command `virtscreen --auto`.

I am informed "Currently Wayland is not supported"
Sorry, I'm not so familiar with the Ubuntu. So I don't know what `Wayland` is. And what's wrong about virtscreen. I don't know what to do. Does this software support Ubuntu 18.04? It would be of great help if you could tell me what to do :)

**Comment by salvager8** [comment]
Hey! searching for an answer to this I've found the solution, IK it was almost two years ago but for the record, wayland is a graphic protocol and a library for linux, so the message says that wayland is not supported is because you are logged into ubuntu using wayland instead of xorg, just log out, go to the settings at the log in screen, select Ubuntu instead of Ubuntu with Wayland and log in again, should be solved :)

this fixed it for ubuntu 20.04, not sure on 22.04 if changing from wayland to xorg is supported

**Comment by nyck33** [comment]
I don't see a way to change @salvager8 on Ubuntu 22.04, I know what you are talking about for Ubuntu 20.04

**Comment by salvager8** [comment]
@nyck33 yes that's right, this response is for ubuntu 20.04, I've updated the answer to reflect this, I didn't update yet to 22.xx so not sure if there's a workaround like this there
